lib/process.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+
+module.exports = {
+  uploadFieldFiles: async function (req, name, files, {
```

Comment:
I would call this `acceptFieldFiles`. The name initially made me think this was browser side code.

Reply:
Or `insertFieldFiles` since we already renamed `accept` to `insert` in the attachment module